Formula/gcal.rb

```diff
@@ -19,7 +19,16 @@ class Gcal < Formula
     sha256 cellar: :any_skip_relocation, yosemite:       "b50b10e57eea6d6fa84c35769401b0da3bde02bade28e137bc1263d492c60c3f"
   end
 
+  uses_from_macos "texinfo" => :build
+
+  on_linux do
+    depends_on "ncurses"
```

Comment:
Why not `uses_from_macos`?

Reply:
There is no linkage to system `ncurses` on macOS, so I figured this notation is more clear about that. Another option is to add a configure flag to the Linux build to disable `ncurses` linkage, if we want feature parity across platforms.